.github/workflows/cpp.yml

```
@@ -55,22 +55,39 @@ env:
 jobs:
   docker:
     name: ${{ matrix.title }}
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.runs-on }}
     if: ${{ !contains(github.event.pull_request.title, 'WIP') }}
     timeout-minutes: 75
     strategy:
       fail-fast: false
       matrix:
-        image:
-          - conda-cpp
-          - ubuntu-cpp-sanitizer
         include:
-          - image: conda-cpp
+          - arch: amd64
+            clang_tools: "14"
```

Could you use `-` not `_` for word separator for `matrix` parameter names like `ruins-on`?

```suggestion
            clang-tools: "14"
```

---

sure, I've done it. Just so I know in the future, is there a reason for it I am not aware of?

---

Because it's GitHub Actions' convention.

---

ok, that makes sense. I found some `_` sometimes but in general they seem to use `-`. There is even a question opened on if the convention exists: https://github.com/community/community/discussions/39547
I'll have it in mind in the future so at least we have a convention. Thanks @kou